src/azure-cli/azure/cli/command_modules/vm/_params.py

```diff
@@ -1322,6 +1322,15 @@ def load_arguments(self, _):
                        'going to be replicated to. This property is updatable. Expected value: '
                        'json-string/json-file/@json-file.')
             c.argument('default_file_link', help='The default configuration link of the artifact, must be a readable storage page blob.')
+
+    with self.argument_context('sig gallery-application create') as c:
+        c.argument('package_file_name', help='The name to assign the downloaded package file on the VM. This is limited to 4096 characters.' 
+                                             'If not specified, the package file will be named the same as the Gallery Application name.')
+        c.argument('config_file_name', help='The name to assign the downloaded config file on the VM. This is limited to 4096 characters. '
+                                            'If not specified, the config file will be named the Gallery Application name appended with "_config"')
+
+    for scope in ['create', 'update']:
+        with self.argument_context('sig gallery-application version {}'.format(scope)) as c:
```

Comment:
```suggestion

    with self.argument_context('sig gallery-application create') as c:
        c.argument('package_file_name', help='The name to assign the downloaded package file on the VM. This is limited to 4096 characters.' 
                                             'If not specified, the package file will be named the same as the Gallery Application name.')
        c.argument('config_file_name', help='The name to assign the downloaded config file on the VM. This is limited to 4096 characters. '
                                            'If not specified, the config file will be named the Gallery Application name appended with "_config"')
```

Reply:
What I understood from this section of the help is that the order in which these arguments are printed are the order in which they get displayed in the help text. I tried to make the order in which the arguments are shown with the help text.

Anyway, I will undo the splitting of the for scope in ['create', 'update'] section, and put the new help text after